integration-tests/environments/react-native-test-app/ios/Podfile

```
@@ -0,0 +1,9 @@
+require_relative '../../../../node_modules/react-native-test-app/test_app'
+
+workspace 'RealmTests.xcworkspace'
+
+options = {
+  :hermes_enabled => ENV['USE_HERMES'] && ENV['USE_HERMES'] == '0' ? false : true,
```

Is `ENV['USE_HERMES'] &&` actually needed here? (Also, looks like if the env is not defined, we default to `true`, is that also the intended behavior?)
```suggestion
  :hermes_enabled => ENV['USE_HERMES'] == '0' ? false : true,
```

---

You can probably simplify this further:
```suggestion
  :hermes_enabled => ENV['USE_HERMES'] != '0',
```

---

You're probably right - I just copied this from the React Native files .. but, we could probably just remove this and upgrade RNTA to the latest version (to get this fix applied https://github.com/microsoft/react-native-test-app/pull/1928).

> we default to true, is that also the intended behavior?

Yep.

---

🙂 👌